packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2193,6 +2193,23 @@ class ShaderMaskLayer extends ContainerLayer {
   }
 }
 
+/// A backdrop key uniquely identifies the backdrop that a [BackdropFilterLayer]
+/// samples from.
+///
+/// When multiple backdrop filters share the same key, the Flutter engine can
+/// more efficiently perform the backdrop operations.
```

Comment:
Suggestion: 
```suggestion
/// Backdrop filters with identical configurations should reuse the same key. 
/// This allows the Flutter engine to potentially optimize the performance of 
/// backdrop operations.
```
This narrates from the user's perspective on what to do and for what purpose.

Reply:
Also, what if the key is used on different backdrops? We might want to mention it.

Reply:
> Also, what if the key is used on different backdrops? 

Can you clarify what you mean here? do you mean different kinds of filters? The implementation handles that and they will only share the filter input and not the processed filter.

Reply:
Oh I think I understand. So is this correct?
```
/// [BackdropFilter]s on the same backdrop should reuse the same key. 
/// This allows the Flutter engine to potentially optimize the performance of 
/// backdrop operations.
```

Reply:
Its beneficial to share backdrop keys even with different filters, as sharing the input can be a pretty dramatic performance improvement. The only scenario you don't want to use them is when the filters overalpy

Reply:
This doc comment is a bit circular because  "the same backdrop" is defined by things that share keys. I think the existing docs are OK for now?

Reply:
Yeah sure!